atlas/commands/bootstrap/bootstrap.go

```diff
@@ -105,6 +108,12 @@ func (b Bootstrap) Run() error {
 		return err
 	}
 
+	if app.WithHelm {
```

Comment:
May be it would be better to make it consistent with  https://github.com/infobloxopen/atlas-cli/blob/master/atlas/application/application.go#L61 ?

Reply:
I thought about this option. In fact, we create files for helm. But also we use an independent application for this. Helm creates not only files, but a complex structure.
Therefore, I thought it would be more correct to place it in the bootstrap package. Moreover, other independent applications are called in this package.